packages/components/src/components/contract-card/contract-card-items/contract-card-body.jsx

```diff
@@ -279,89 +56,117 @@ const ContractCardBody = ({
         />
     );
 
-    const card_body = is_multiplier ? (
-        <MultiplierCardBody
-            addToast={addToast}
-            connectWithContractUpdate={connectWithContractUpdate}
-            contract_info={contract_info}
-            contract_update={contract_update}
-            currency={currency}
-            current_focus={current_focus}
-            error_message_alignment={error_message_alignment}
-            getCardLabels={getCardLabels}
-            getContractById={getContractById}
-            has_progress_slider={has_progress_slider}
-            progress_slider={progress_slider_mobile_el}
-            is_mobile={is_mobile}
-            is_sold={is_sold}
-            onMouseLeave={onMouseLeave}
-            status={status}
-            removeToast={removeToast}
-            setCurrentFocus={setCurrentFocus}
-            should_show_cancellation_warning={should_show_cancellation_warning}
-            toggleCancellationWarning={toggleCancellationWarning}
-        />
-    ) : is_vanilla ? (
-        <VanillaOptionsCardBody
-            contract_info={contract_info}
-            currency={currency}
-            getCardLabels={getCardLabels}
-            is_sold={is_sold}
-            has_progress_slider={has_progress_slider}
-            progress_slider={progress_slider_mobile_el}
-            status={status}
-        />
-    ) : (
-        <React.Fragment>
-            <div className='dc-contract-card-items-wrapper'>
-                <ContractCardItem
-                    header={is_sold ? getCardLabels().PROFIT_LOSS : getCardLabels().POTENTIAL_PROFIT_LOSS}
-                    is_crypto={isCryptocurrency(currency)}
-                    is_loss={+profit < 0}
-                    is_won={+profit > 0}
-                >
-                    <Money amount={profit} currency={currency} />
-                    <div
-                        className={classNames('dc-contract-card__indicative--movement', {
-                            'dc-contract-card__indicative--movement-complete': is_sold,
-                        })}
-                    >
-                        {status === 'profit' && <Icon icon='IcProfit' />}
-                        {status === 'loss' && <Icon icon='IcLoss' />}
-                    </div>
-                </ContractCardItem>
-                <ContractCardItem header={is_sold ? getCardLabels().PAYOUT : getCardLabels().INDICATIVE_PRICE}>
-                    <Money currency={currency} amount={sell_price || indicative} />
-                    <div
-                        className={classNames('dc-contract-card__indicative--movement', {
-                            'dc-contract-card__indicative--movement-complete': is_sold,
-                        })}
+    let card_body;
```

Comment:
@farzin-deriv Could you please elaborate in what way exactly we will benefit from it? if it remains inside the component, it doesn't matter if we assign a certain `XxxxCardBody` to a current `card_body` variable or to a new `Body` variable that you suggest instead. Otherwise, if we put this Body outside into a separate component, we will pass all the props required for all 5 variations (including accumulators), and we'll write it twice because this part of code is used twice in this file at L198 and L208. Cards for specific trade types are already separated from this component even though this refactoring can be done after release. We can also think about general improvement of this file later when the project deadline is not that tough. Can we?🙂